tasks/level-1/2.2.1.2.yml

```diff
@@ -1,33 +0,0 @@
-# Standards: 0.11
```

Comment:
Should we leave this in place and simply include it in the exclusions if we don't want it?

Reply:
for some reason @alpozcan is unable to respond to comments.

Tracking discussion from other channels.

> It was removed because a) it is no longer included in the benchmark b) it belongs to the section-2 branch anyway. The only reason I ever delete something is when it no longer appears in the benchmark - but sometimes it just changes section numbers (we will see a lot of these in section-2 PR)